src/Servers/IIS/IIS/src/Core/IISHttpServer.cs

```diff
@@ -135,7 +135,7 @@ private static NativeMethods.REQUEST_NOTIFICATION_STATUS HandleRequest(IntPtr pI
                 // Unwrap the server so we can create an http context and process the request
                 server = (IISHttpServer)GCHandle.FromIntPtr(pvRequestContext).Target;
 
-                var context = server._iisContextFactory.CreateHttpContext(pInProcessHandler);
+                var context = server?._iisContextFactory.CreateHttpContext(pInProcessHandler);
```

Comment:
This null check looks bogus. You'll get a null context back then an exception will be thrown when you QUWI(null)